pandas_gbq/tests/test_gbq.py

```diff
@@ -64,6 +56,15 @@ def _in_travis_environment():
            'GBQ_PROJECT_ID' in os.environ
 
 
+def _get_dataset_id():
+    if _in_travis_environment():
+        python_version = os.environ.get('PYTHON').replace('.', '_')
```

Comment:
maybe you should just generate an id here instead (say 10 random characters or something). this might collide if multiple travis workers are running (e.g. 2 PR's)